integration-tests/tests/api/otel-collector.spec.ts

```diff
@@ -0,0 +1,431 @@
+import { clickHouseQuery } from '../../testkit/clickhouse';
+import { graphql } from '../../testkit/gql';
+import { ResourceAssignmentModeType } from '../../testkit/gql/graphql';
+import { execute } from '../../testkit/graphql';
+import { initSeed } from '../../testkit/seed';
+import { getServiceHost } from '../../testkit/utils';
+
+const TargetTracesQuery = graphql(`
+  query TargetTracesQuery($targetId: ID!, $traceIds: [ID!]) {
+    target(reference: { byId: $targetId }) {
+      id
+      viewerCanAccessTraces
+      traces(first: 10, filter: { traceIds: $traceIds }) {
+        edges {
+          node {
+            id
+            operationName
+            operationType
+            success
+            httpStatusCode
+            httpMethod
+            spans {
+              name
+            }
+          }
+        }
+      }
+    }
+  }
+`);
+
+async function sendTrace(args: {
```

Comment:
Does this function belong in our testkit?

Reply:
Could you please elaborate?